app/views/declaration/additional_information.scala.html

```diff
@@ -49,19 +49,19 @@
                 <table id="additional_information">
                     <thead>
                         <tr>
-                            <th class="code-header" colspan="1">@messages("supplementary.additionalInformation.table.headers.code")</th>
-                            <th class="description-header" colspan="1">@messages("supplementary.additionalInformation.table.headers.description")</th>
-                            <th colspan="1"></th>
+                            <th class="code-header" scope="col">@messages("supplementary.additionalInformation.table.headers.code")</th>
+                            <th class="description-header" scope="col">@messages("supplementary.additionalInformation.table.headers.description")</th>
+                            <td></td>
                         </tr>
                     </thead>
                     <tbody>
                     @for((elem, index) <- items.zipWithIndex) {
                         <tr id="removable_elements__row@index">
-                            <th class="code-row" id="removable_elements-row@index-code">@{elem.code}</th>
-                            <th class="description-row" id="removable_elements-row@index-info">@{elem.description}</th>
-                            <th id="removable_elements-row@index-remove_button">
+                            <td class="code-row" id="removable_elements-row@index-code" scope="row">@{elem.code}</td>
```

Comment:
If it was a `th` before should we add `bold` class to this?

Reply:
**TH** is used for table header cells while **TD** is used for table data cells. This distinction gives user agents a means to render such cells distinctly, for instance by using a larger or heavier font for header cells.